aruco_tracking:
  process_freq: 10.0
  
  # flag
  is_eval: false
  compare_mode: false

  # threshold
  marker_cnt_th: 25 # count for fixing target when multi marker detected
  filter_buf_size: 10 # moving average filter buff size
  noise_dist_th_m: 3.0
  noise_cnt_th: 6 # (3/5)*process_frequency

  # aruco marker
  dictionaryID: 0
  big_marker_id: 0
  small_marker_id: 1
  big_marker_size_m: 0.2
  small_marker_size_m: 0.06